tutorials/R/spatial-data/spatialData.Rmd

```diff
@@ -1,14 +1,15 @@
 ---
 syncID: b152963c4883463883c3b6f0de95fd93
 title: "Access and Work with NEON Geolocation Data"
-description: "Use files available on the data portal, the NEON API, and the neonUtilities R package to access the locations of NEON sampling events and infrastructure. Calculate more precise locations for certain sampling types, and reference ground sampling to airborne data."
+description: "Use files available on the NEON data portal, NEON API, and  neonUtilities R package to access the locations of NEON sampling events and infrastructure. Calculate more precise locations for certain sampling types and reference ground sampling to airborne data."
 dateCreated: 2019-09-13
 authors: [Claire K. Lunch]
-contributors:
+contributors: Megan Jones
 estimatedTime: 40 minutes
 packagesLibraries: neonUtilities
 topics: data-management, rep-sci
 languageTool: R
+dataProduct: 
```

Comment:
Added this back in as we're considering if a "all" is appropriate for mapping to data product pages for those tutorials that apply to all data products.